recipes/loos/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# Note: there are many handy hints in comments in this example -- remove them
+# when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only
+# here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+# {% set name = "loos" %}
+# {% set version = "3.2beta" %}
+{% set version = "3.2" %}
+
+package:
+  name: "loos"
+  version: "{{  version  }}"
+
+source:
+  url: https://raw.githubusercontent.com/GrossfieldLab/loos/master/archive/loos-{{  version  }}.tar.gz
+  sha256: 13a752678f98e5263595715f9cb145976fe52695d82b634f4f43b9f52b784701
+
+build:
+  number: 0
+  # Only python 3.x
+  skip: True  # [win]
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - scons
+  host:
+    - python
+    - scons
+    - swig
+    - numpy
+    - scipy
+    - boost
+    - openblas
+    - libnetcdf
+    - lapack
+    - eigen
+    - openssl
+  run:
+    - python
+    - {{  pin_compatible('numpy')  }}
+    - {{  pin_compatible('scipy')  }}
+test:
+  imports:
+    - loos
+
+about:
+  home: https://github.com/GrossfieldLab/loos
+  license: GPLv3
```

Comment:
Choose `GPL-3.0-only` or `GPL-3.0-or-later` for the `license`, whichever is appropriate